### DISCUSSION TITLE
Bloatynosy Nue 0.88

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#247] [Announcements]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Bloatynosy Nue! The king is back... The original repo has been revived and is now online <a href="https://t.co/iCiPTbYhlw">https://t.co/iCiPTbYhlw</a> <a href="https://t.co/qo8FbL9uTj">pic.twitter.com/qo8FbL9uTj</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1857164868954325081?ref_src=twsrc%5Etfw">November 14, 2024</a></blockquote>

- The design has been further simplified and modernized for a smoother user experience. Additionally, the file size and resource usage have been optimized, making Bloatynosy a true lightweight in every sense of the word
- A new plugin store has been added, allowing you to download and install plugins regardless of your version. The store will also support third-party scripts in the future, but most available plugins are developed by me. For example, you’ll find [Flyby11 scripts for upgrading to Windows 11, including in-place upgrades using server setups](https://github.com/builtbybel/Flyby11)
- The built-in app removal tool Dumputer now features two "baskets," allowing you to drag bloatware apps from one box to the second trash bin. This UI design is inspired by my old app Bloatbox—fun fact, the classic Bloatynosy version 0.85 had a similar concept
-  Managing plugins is easier than ever. A single click on "Visit Store" gives you access to the store and allows you to enable or explore new plugins effortlessly

![Bloatynosy_Nue_OMlHOHBkAK](https://github.com/user-attachments/assets/6d310ec4-81d3-431a-b514-cc1830062097)

Some of you want to thank me, so [I’ll drop my PayPal link here](https://paypal.com/donate?hosted_button_id=MY7HX4QLYR4KG). Thanks a lot, friends, and see you soon!

<details>
  <summary>If you haven’t heard yet, BloatyNosy has been brought back to life, read on to learn more...</summary>

You probably remember my classic BloatynosyApp, which had to go through some renaming and eventually got discontinued. Everything that came after, like the web-based "Winpilot," didn't last long either. Honestly, I thought I was done with Bloatynosy-well, at least with the name. But then, during a recent visit to a well-known company, I saw an employee Googling the app and trying to download it, and I had to stifle a laugh. Apparently, the legend lives on. The employee and I had a good laugh about it, and he even called it the epitome of debloating.
Sometimes, I think a little too long about things, and then a useful project just pops out. Bloatynosy was one of those projects. Sure, the name wasn't for everyone, but it was the go-to tool for cleaning up and debloating Windows. Recently, I added an OOBE assistant called "Fresh11" to the FlybyScript—basically, an all-in-one solution and I figured, why not use the same foundation to bring Bloatynosy back to life?

I had a hard time committing to it, but in the end, I went for it. Now I can proudly say that after Bloatynosy, there won’t be anything newer or better. This time, it'll be even leaner, more efficient, smaller, and faster. A simple .exe that anyone can understand, doing exactly what bugs us after installing Windows 11. **The reborn Bloatynosy carries the "Nue" extension (Bloatynosy Nue)**, not "New" ;) The last solid, classic version was 0.85, and that's where Im picking up (with a fresh new interface). The build carries a small 11.x version number.

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Bloatynosy/releases/tag/0.88.0'>Bloatynosy Nue 0.88</a>.</em>

**Comment by mobyFrag** [comment]
Love the new design! now fully integrate in Windows 11 environment 😍
Thanks Belim!

tiny glitch: icon and text are not centered here :
![image](https://github.com/user-attachments/assets/bb2e49e4-30da-48a5-9d2f-e246f9c837d0)

**Reply by Belim** [reply]
yeah **Beauty**nosy 🤣

**Comment by NoName-coding** [comment]
probably missed a step to install but the .Exe isn't working for me, tried putting it straight up on my desktop, tried to dl the source and to put the .exe inside, etc etc, any help?
